### PR TITLE
Q3 2020 update

### PR DESCRIPTION
**Pull request by gregestren**

### CODE REVIEW

File: roadmaps/configuration.md [hunk 1]
```diff
@@ -135,35 +140,43 @@ impact of configuration APIs**
...
+<span class="etabox">Unclear ETA</span>**An experimental Bazel mode automatically
 "trims" build graphs**
 <span class="notstartedstatus">NOT STARTED</span> ([#6524](https://github.com/bazelbuild/bazel/issues/6524))
 
 * e.g. C++ rules don't re-analyze because of Java flag changes
 * Building a binary's *Mac* and *Linux* versions simultaneously doesn't double the build graph
 
 <div class="padbottom"></div>
-<span class="etabox">Jul 2020</span>**An experimental Bazel mode caches
+<span class="etabox">Oct 2020</span>**An experimental Bazel mode caches
```
Comment (aiuto) [inline roadmaps/configuration.md:151]: why don't we spell out the months.  That is probably nicer for non-native english speakers.

**Comment by laurentlb** [comment]
Thanks for keeping it up-to-date!

File: roadmaps/configuration.md [hunk 2]
```diff
@@ -135,35 +140,43 @@ impact of configuration APIs**
...
+* Proof-of-concepts a [more general
   design](https://docs.google.com/document/d/17snvmic26-QdGuwVw55Gl0oOufw9sCVuOAvHqGZJFr4/edit)
   that guarantees ideal caching for *all* actions
+* *July update*: This is being distinctly up-prioritized with a ~3 month ETA.
 
 
 ### Features
 
 <div class="padbottom"></div>
-<span class="etabox">Mar 2020</span>**Projects can declare and compliance-check software
```
Comment (gregestren) [inline roadmaps/configuration.md:159]: @aiuto how would you suggest we update this?

File: roadmaps/configuration.md (hunk 1 above)
Comment (gregestren) [reply roadmaps/configuration.md]: Done.

File: roadmaps/configuration.md (hunk 2 above)
Comment (aiuto) [reply roadmaps/configuration.md:159]: This may or may not happen in Q3.  I expect I'll have it done by late fall. How about saying Q4

**Pull request merged**